plugins/mcreator-core/templates/animations/constant_slow_x_rotation.json

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
No need for object here as you are just listing strings, so JSON array is enough:

```json
[
    ".rotateAngleX = f2 / 20.f;"
]
```